api/routes/analysis.py

```diff
@@ -238,16 +238,16 @@ async def query_analyses(
 
 @router.get('/analysis-runner', operation_id='getAnalysisRunnerLog')
 async def get_analysis_runner_log(
-    project_names: list[str] = Query(None),  # type: ignore
+    project_names: Annotated[list[str], Query()] = [],  # type: ignore
```

Comment:
Can we remove `type: ignore` here?

Reply:
Switched to `noqa` due to the mutable list thing. I added a doc link above